packages/performance/src/services/perf_logger.test.ts

```diff
@@ -108,12 +113,136 @@ describe('Performance Monitoring > perf_logger', () => {
 
     it('does not log an event if cookies are disabled in the browser', () => {
       stub(Api.prototype, 'requiredApisAvailable').returns(false);
+      stub(attributeUtils, 'getVisibilityState').returns(VISIBILITY_STATE);
       const trace = new Trace(TRACE_NAME);
       trace.record(START_TIME, DURATION);
       clock.tick(1);
 
       expect(addToQueueStub).not.to.be.called;
     });
+
+    it('Amount of customMetric is 32.', () => {
+      const EXPECTED_TRACE_MESSAGE =
+        `{` +
+        webAppInfo +
+        `,"trace_metric":{"name":"${TRACE_NAME}","is_auto":false,\
+"client_start_time_us":${START_TIME * 1000},"duration_us":${DURATION * 1000},\
+"counters":{"counter1":1,"counter2":2,"counter3":3,"counter4":4,"counter5":5,"counter6":6,\
+"counter7":7,"counter8":8,"counter9":9,"counter10":10,"counter11":11,"counter12":12,\
+"counter13":13,"counter14":14,"counter15":15,"counter16":16,"counter17":17,"counter18":18,\
+"counter19":19,"counter20":20,"counter21":21,"counter22":22,"counter23":23,"counter24":24,\
+"counter25":25,"counter26":26,"counter27":27,"counter28":28,"counter29":29,"counter30":30,\
+"counter31":31,"counter32":32}}}`;
+      getIidStub.returns(IID);
+      stub(attributeUtils, 'getVisibilityState').returns(VISIBILITY_STATE);
+      SettingsService.getInstance().loggingEnabled = true;
+      SettingsService.getInstance().logTraceAfterSampling = true;
+      const trace = new Trace(TRACE_NAME);
+      for (let i = 1; i <= 32; i++) {
+        trace.putMetric('counter' + i, i);
+      }
+      trace.record(START_TIME, DURATION);
+      clock.tick(1);
+
+      expect(addToQueueStub).to.be.called;
+      expect(addToQueueStub.getCall(0).args[0].message).to.be.equal(
+        EXPECTED_TRACE_MESSAGE
+      );
+    });
+
+    it('Amount of customAttribute is 5.', () => {
```

Comment:
same as above.

Reply:
Changed it to align with the rest of the test names.